app/src/main/res/layout-land/item_artist.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
I do not think you need ConstraintLayout here. Lets replace it with FrameLayout for better performance?